### PR TITLE
handle read-only databases separately

### PR DESCRIPTION
**Pull request by snopoke**
buddy @kaapstorm 

along with https://github.com/dimagi/commcare-hq/pull/17643

### CODE REVIEW

**Comment by snopoke** [comment]
Deployed this to all envs so HQ change is GTG

File: ansible/vars/icds/icds_public.yml [hunk 1]
```diff
@@ -135,10 +135,12 @@ postgresql_dbs:
   - name: commcarehq_ucr
     host: "{{ groups.postgresql.0 }}"
     django_alias: ucr
+    django_migrate: False
   - django_alias: icds-ucr
     name: commcarehq_icds_aggregatedata
     host: "{{ groups.postgresql.0 }}"
     query_stats: True
+    django_migrate: False
```
Comment (emord) [inline ansible/vars/icds/icds_public.yml]: @snopoke It's possible that we do want to migrate this https://github.com/dimagi/commcare-hq/tree/master/custom/icds_reports/migrations

I think we have moved most of the table generation into sql scripts that run druing the aggregation, but historically there have been migrations on icds-ucr. @sheelio can you confirm

**Comment by sheelio** [comment]
Yes, we have SQL Scripts that have migrated those tables (mainly adding or
removing columns).

File: ansible/vars/icds/icds_public.yml (hunk 1 above)
Comment (snopoke) [reply ansible/vars/icds/icds_public.yml]: abf9e86

**Pull request merged**